src/freenas/etc/system_info

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
Not sure how useful it would be to have a system wide shell library like this, but if we choose this path, it should be somewhere inside /usr/local/share, not /etc, IMO.